src/Codeception/Lib/Connector/Mezzio.php

```diff
@@ -34,11 +39,9 @@ class Mezzio extends Client
 
     /**
      * @param BrowserKitRequest $request
-     *
-     * @return Response
-     * @throws \Exception
+     * @throws Exception
      */
-    public function doRequest($request)
+    public function doRequest($request): Response
```

Comment:
I'd love this, but it's a BC Break with little advantage over the already present DocBlock.
So please leave public and protected methods as is, and only update private ones.

Reply:
@Slamdunk Adding return types is really what is motivating me to submit this PR.

Initially, I submitted [a PR](https://github.com/Codeception/module-symfony/pull/131) to allow chained methods through a fluent interface, and I ran [into this error.](https://github.com/Codeception/Codeception/issues/6164) If you look at that thread you will understand why this change is necessary, and why this BC is intentional.

The idea is to release this as a major version of the module. in the case of Mezzio it would be `3.0.0`, which is also consistent with [the improvements](https://github.com/Codeception/Codeception/pull/6137) included in Codeception 5.

Reply:
@Slamdunk I undid this change specifically since `doRequest` comes [from here](https://github.com/symfony/browser-kit/blob/d6034732ee75aa06c1a28cbbd94a767abed88ae4/AbstractBrowser.php#L240) and there are no return types there yet.

Nevertheless, Adding return types is really what is motivating me to submit this PR. If you look at [this thread](https://github.com/Codeception/Codeception/issues/6164) you will understand why this BC is intentional.

The idea is to release this as a major version of the module. In the case of Mezzio it would be `3.0.0`, which is also consistent with the improvements [included](https://github.com/Codeception/Codeception/pull/6137) in Codeception 5.